packages/opc-back-to-top/README.md

```diff
@@ -0,0 +1,127 @@
+# This is opc-back-to-top 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/
+
+A standardized web component based on Lit element for Red Hat One Platform back to top.
+
+## Install
+
+```sh
+npm install
+```
+
+## Usage
+plain back to top
+```html
+<opc-back-to-top></opc-back-to-top>
+```
+Back-to-top with reveal-at attribute, It reveals back to top once user scrolls upto specified pixels downwards.
+```html
+<opc-back-to-top reveal-at="200"></opc-back-to-top>
+```
+
+Back to top with slot, In slot users can pass icon and text of their choice.
```

Comment:
this should be written as `back-top-top with default slot (optional) where users can pass icon and text if they like`